modules/oadp-pod-crash-set-resource-request-velero.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="oadp-pod-crash-resource-request-velero_{context}"]
+= Setting resource requests for a Velero pod
+
+You can use the `configuration.velero.podConfig.resourceAllocations` specification field in the `oadp_v1alpha1_dpa.yaml` file to set specific resource requests for a `Velero` pod.
+
+.Procedure
+
+* Set the `cpu` and `memory` resource requests in the `yaml` file:
+
```

Comment:
This is missing a `+` to keep the code block with the step in the procedure

Reply:
Fixed.